easybuild/easyconfigs/i/IPython/IPython-8.14.0-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'PythonBundle'
+
+name = 'IPython'
+version = '8.14.0'
+
+homepage = 'https://ipython.org/index.html'
+description = """IPython provides a rich architecture for interactive computing with:
+ Powerful interactive shells (terminal and Qt-based).
+ A browser-based notebook with support for code, text, mathematical expressions, inline plots and other rich media.
+ Support for interactive data visualization and use of GUI toolkits.
+ Flexible, embeddable interpreters to load into your own projects.
+ Easy to use, high performance tools for parallel computing."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+builddependencies = [
+    ('binutils', '2.40'),
+]
+dependencies = [
+    ('Python', '3.11.3'),
+    ('ZeroMQ', '4.3.4'),
+    ('lxml', '4.9.2'),
+    ('BeautifulSoup', '4.12.2'),
+]
+
+sanity_pip_check = True
+use_pip = True
+
+# for the matplotlib-inline required extention we avoid the import sanity check
+# as it will fail without matplotlib in the environment, but ipython devs prefer not to make
+# matplotlib a required dep (https://github.com/ipython/matplotlib-inline/issues/4)
+# we follow the same convention and we not set matplotlib as dependency
+
+exts_list = [
+    ('traitlets', '5.9.0', {'checksums': ['f6cde21a9c68cf756af02035f72d5a723bf607e862e7be33ece505abf4a3bad9']}),
+    ('pure_eval', '0.2.2', {'checksums': ['2b45320af6dfaa1750f543d714b6d1c520a1688dec6fd24d339063ce0aaa9ac3']}),
+    ('executing', '1.2.0', {'checksums': ['19da64c18d2d851112f09c287f8d3dbbdf725ab0e569077efb6cdcbd3497c107']}),
+    ('asttokens', '2.2.1', {'checksums': ['4622110b2a6f30b77e1473affaa97e711bc2f07d3f10848420ff1898edbe94f3']}),
+    ('stack_data', '0.6.2', {'checksums': ['32d2dd0376772d01b6cb9fc996f3c8b57a357089dec328ed4b6553d037eaf815']}),
+    ('prompt_toolkit', '3.0.39', {'checksums': ['04505ade687dc26dc4284b1ad19a83be2f2afe83e7a828ace0c72f3a1df72aac']}),
+    ('pickleshare', '0.7.5', {'checksums': ['87683d47965c1da65cdacaf31c8441d12b8044cdec9aca500cd78fc2c683afca']}),
+    ('matplotlib-inline', '0.1.6', {'modulename': False, 'checksums': ['f887e5f10ba98e8d2b150ddcf4702c1e5f8b3a20005eb0f74bfdbd360ee6f304']}),
+    ('parso', '0.8.3', {'checksums': ['8c07be290bb59f03588915921e29e8a50002acaf2cdc5fa0e0114f91709fafa0']}),
+    ('jedi', '0.18.2', {'checksums': ['bae794c30d07f6d910d32a7048af09b5a39ed740918da923c6b780790ebac612']}),
+    ('backcall', '0.2.0', {'checksums': ['5cbdbf27be5e7cfadb448baf0aa95508f91f2bbc6c6437cd9cd06e2a4c215e1e']}),
+    ('ipython', '8.14.0', {'modulename': 'IPython', 'checksums': ['1d197b907b6ba441b692c48cf2a3a2de280dc0ac91a3405b39349a50272ca0a1']}),
+]
```

Comment:
Suggestion to make the check happy and update `jedi` to `0.19.0`:
```suggestion
exts_list = [
    ('traitlets', '5.9.0', {
        'checksums': ['f6cde21a9c68cf756af02035f72d5a723bf607e862e7be33ece505abf4a3bad9'],
    }),
    ('pure_eval', '0.2.2', {
        'checksums': ['2b45320af6dfaa1750f543d714b6d1c520a1688dec6fd24d339063ce0aaa9ac3'],
    }),
    ('executing', '1.2.0', {
        'checksums': ['19da64c18d2d851112f09c287f8d3dbbdf725ab0e569077efb6cdcbd3497c107'],
    }),
    ('asttokens', '2.2.1', {
        'checksums': ['4622110b2a6f30b77e1473affaa97e711bc2f07d3f10848420ff1898edbe94f3'],
    }),
    ('stack_data', '0.6.2', {
        'checksums': ['32d2dd0376772d01b6cb9fc996f3c8b57a357089dec328ed4b6553d037eaf815'],
    }),
    ('prompt_toolkit', '3.0.39', {
        'checksums': ['04505ade687dc26dc4284b1ad19a83be2f2afe83e7a828ace0c72f3a1df72aac'],
    }),
    ('pickleshare', '0.7.5', {
        'checksums': ['87683d47965c1da65cdacaf31c8441d12b8044cdec9aca500cd78fc2c683afca'],
    }),
    ('matplotlib-inline', '0.1.6', {
        'modulename': False,
        'checksums': ['f887e5f10ba98e8d2b150ddcf4702c1e5f8b3a20005eb0f74bfdbd360ee6f304'],
    }),
    ('parso', '0.8.3', {
        'checksums': ['8c07be290bb59f03588915921e29e8a50002acaf2cdc5fa0e0114f91709fafa0'],
    }),
    ('jedi', '0.19.0', {
        'checksums': ['bcf9894f1753969cbac8022a8c2eaee06bfa3724e4192470aaffe7eb6272b0c4'],
    }),
    ('backcall', '0.2.0', {
        'checksums': ['5cbdbf27be5e7cfadb448baf0aa95508f91f2bbc6c6437cd9cd06e2a4c215e1e'],
    }),
    ('ipython', version, {
        'modulename': 'IPython',
        'checksums': ['1d197b907b6ba441b692c48cf2a3a2de280dc0ac91a3405b39349a50272ca0a1'],
    }),
]
```

Reply:
Done